app/src/main/java/org/oppia/android/app/topic/revisioncard/RevisionCardFragmentPresenter.kt

```diff
@@ -79,6 +86,45 @@ class RevisionCardFragmentPresenter @Inject constructor(
     return binding.root
   }
 
+  private fun setupRevisionNavigationCards(
```

Comment:
```suggestion
  private fun setUpRevisionNavigationCards(
```

Nit: verb vs. noun form.

Reply:
done